tensorflow/lite/kernels/slice_test.cc

```diff
@@ -238,6 +238,28 @@ TEST_P(SliceOpTest, SliceString) {
                                 "2,0,0,0", "2,0,1,0", "2,0,2,0"}));
 }
 
+TEST(SliceOpTest, SliceInt64) {
+  SliceOpModel<int64_t, int32_t> m({3, 2, 3, 1}, {4}, {4}, TensorType_INT32,
+                                   TensorType_INT64);
+  m.SetInput({1, 1, 1, 2, 2, 2, 3, 3, 3, 4, 4, 4, 5, 5, 5, 6, 6, 6});
+  m.SetBegin({1, 0, 0, 0});
+  m.SetSize({2, 1, -1, 1});
+  m.Invoke();
+  EXPECT_THAT(m.GetOutputShape(), ElementsAreArray({2, 1, 3, 1}));
+  EXPECT_THAT(m.GetOutput(), ElementsAreArray({3, 3, 3, 5, 5, 5}));
+}
+
+TEST(SliceOpTest, SliceBool) {
+  SliceOpModel<bool, int32_t> m({2, 3}, {2}, {2}, TensorType_INT32,
```

Comment:
`tensorflow/lite/kernels/slice_test.cc:253:31: error: no matching constructor for initialization of 'SliceOpModel<bool, int32_t>' (aka 'SliceOpModel<bool, int>')
  SliceOpModel<bool, int32_t> m({2, 3}, {2}, {2}, TensorType_INT32,`